src/BalanceTx.purs

```diff
@@ -413,6 +414,44 @@ _redeemersTxIns = lens' \(UnattachedUnbalancedTx rec@{ redeemersTxIns }) ->
   redeemersTxIns /\
     \rdmrs -> UnattachedUnbalancedTx rec { redeemersTxIns = rdmrs }
 
+--------------------------------------------------------------------------------
+-- Setting collateral, collateral return, total collateral
+--------------------------------------------------------------------------------
+
+setCollateral
+  :: Transaction -> Utxo -> QueryM (Either BalanceTxError Transaction)
+setCollateral transaction utxos =
```

Comment:
The variable name is in plural form, but the type is in singular, could you please change either?